src/utilities/general.c

```diff
@@ -73,6 +73,9 @@ hypre_HandleDestroy(hypre_Handle *hypre_handle_)
    hypre_HandleDeviceData(hypre_handle_) = NULL;
 #endif
 
+   /* Deallocate error messages in error handler */
```

Comment:
I think a comment here about memory makes sense because we are in a destroy routine.

Reply:
Sounds good!